### ISSUE TITLE
Python3 compatibility?

### ISSUE THREAD

**Issue by dspreitz** [Turbo87/flarmnet.py#1]
I tried to use the flarmnet db with python3 in following code:
```
import flarmnet
with open('data.fln', 'r') as ffile:
    reader = flarmnet.Reader(ffile)
    for record in reader.read():
        print(record.id)
```

This resulted in following output:
```
(venv) MacBook:ogn-silentwings DSpreitz$ python3 tester2.py 
Traceback (most recent call last):
  File "tester2.py", line 5, in <module>
    print(record.id)
  File "/Users/DSpreitz/ogn-silentwings/venv/lib/python3.6/site-packages/flarmnet/record.py", line 20, in id
    return self.get_or_cache('id', 0, 12)
  File "/Users/DSpreitz/ogn-silentwings/venv/lib/python3.6/site-packages/flarmnet/record.py", line 10, in get_or_cache
    self._cache[attribute] = decrypt(self._line[start:end])
  File "/Users/DSpreitz/ogn-silentwings/venv/lib/python3.6/site-packages/flarmnet/encryption.py", line 3, in decrypt
    for i in xrange(0, len(input), 2):
NameError: name 'xrange' is not defined
```

Is this a python3 compatibility issue? Or did the flarmnet file format change?

**Comment by Turbo87** [comment]
> Is this a python3 compatibility issue?

yes, should be relatively easy to fix though. we can probably replace `xrange()` with `range()`, which should in Python 2 **and** 3

**Comment by Turbo87** [comment]
Resolved by #2
